src/services/rns/rns.precache.ts

```diff
@@ -57,7 +57,11 @@ export async function processRskOwner (eth: Eth, logger: Logger, contractAbi: Ut
     const name = Utils.hexToAscii('0x' + decodedData.params[2].value.slice(218, decodedData.params[2].value.length))
     const tokenId = Utils.numberToHex(Utils.sha3(name) as string)
     const ownerAddress = rskOwnerEvent.returnValues.to.toLowerCase()
-    await Domain.upsert({ tokenId, name: `${name}.${tld}` })
+    try {
+      await Domain.upsert({ tokenId, name: `${name}.${tld}` })
+    } catch (e) {
+      logger.warn(`Domain name ${name}.${tld} for token ${tokenId} could not be stored.`)
```

Comment:
I think we should UPSERT with null or empty name, so we dont lose the relationship. Not just log the issue.

Reply:
I think we are using NULL for domains that we could not retrieve the"readable" name